autoscale_cloudroast/test_repo/autoscale/fixtures.py

```diff
@@ -214,6 +238,10 @@ def setUpClass(cls):
         cls.lbaas_client = lbaas_client
         cls.rcv3_client = rcv3_client
 
+        cls.lc_image_ref, cls.lc_image_ref_alt = fetch_image_ids(images_client)
```

Comment:
Pretty much. Honestly, I wasn't even thinking about mutable state / better patterns when doing this. I just wanted to get this done to move forward.